layouts/default.vue

```diff
@@ -1,46 +1,51 @@
 <template>
   <div>
-    <Header />
-    <Nuxt />
-    <Donate />
-    <Footer />
+    <HeaderComponent />
+    <div class="page">
+      <Nuxt />
+    </div>
+    <DonateComponent />
+    <FooterComponent />
   </div>
 </template>
 
-<style src="@/assets/style.scss" lang="scss">
-</style>
-
 <script>
-import Header from '@/components/header'
-import Donate from '@/components/donate'
-import Footer from '@/components/footer'
+import HeaderComponent from '@/components/header'
+import DonateComponent from '@/components/donate'
+import FooterComponent from '@/components/footer'
 
 export default {
   name: 'App',
   components: {
-    Header,
-    Donate,
-    Footer
-  },
+    HeaderComponent,
+    DonateComponent,
+    FooterComponent
+  }/* ,
   mounted () {
-    this.getApiStatus()
+    this.getApiStatus()// Masel says API check is redundant now
   },
   methods: {
     async getApiStatus () {
-      const apiEl = document.getElementById('js-api-status')
+      const element = document.getElementById('api-status')
 
       try {
         const quaggans = await fetch('https://api.guildwars2.com/v2/quaggans')
 
         if (quaggans.status >= 200 && quaggans.status < 400) {
-          apiEl.textContent = 'API is Online! ☑'
+          element.innerHTML = 'API is Online!'
+          element.className = 'success'
         } else {
-          apiEl.textContent = 'Error reaching API ☒'
+          element.innerHTML = 'Error reaching API'
+          element.className = 'failure'
         }
       } catch {
-        apiEl.textContent = 'Error reaching API ☒'
+        element.innerHTML = 'Error reaching API'
+        element.className = 'failure'
       }
     }
-  }
+  } */
```

Comment:
Should be deleted if its redundant

Reply:
You're absolutely right - I actually left this comment for you to find, so you would understand why the API check was removed, so I would not need to remember and explain in case you asked lol! Thanks I will remove